src/main/java/org/greenbuttonalliance/gbaresourceserver/common/model/DateTimeInterval.java

```diff
@@ -14,21 +14,23 @@
  *  limitations under the License.
  */
 
-package org.greenbuttonalliance.gbaresourceserver.usage.web.dto;
+package org.greenbuttonalliance.gbaresourceserver.common.model;
```

Comment:
The way GitHub is showing these changes is a bit confusing; the old file was the `DateTimeInterval` class used with the DTOs, but the new file is the embeddable entity class. We moved it to a new `common` package since it'll be used by both energy-usage and retail-customer entities.